ext/json/json.c

```diff
@@ -177,6 +181,18 @@ static const char *php_json_get_error_msg(php_json_error_code error_code) /* {{{
 }
 /* }}} */
 
+static zend_string *php_json_get_error_msg_with_location(php_json_error_code error_code, size_t line, size_t column) /* {{{ */
+{
+	const char *base_msg = php_json_get_error_msg(error_code);
+	
+	if (line > 0 && column > 0) {
+		return zend_strpprintf(0, "%s near location %zu,%zu", base_msg, line, column);
```

Comment:
```suggestion
		return zend_strpprintf(0, "%s at line %zu:%zu", base_msg, line, column);
```

`line:column` is industry standard formatting